examples/canvas/canvas/app.py

```diff
@@ -94,6 +104,16 @@ def startup(self):
                         self.dash_pattern_selection
                     ]
                 ),
+                toga.Box(
+                    style=Pack(direction=ROW),
+                    children=[
+                        toga.Label("X Translate:"),
```

Comment:
This text is cut for some reason.

Reply:
That's... odd. From some experimentation, it looks like it's a sizing problem. If the hinted size of the label is 1px larger, it fits. I'll put in that extra pixel a hack for now, but this is something we should probably look at. It feels like it might be a Travertino problem, doing a >= rather than > somewhere...